Classes/Controller/GalleryController.php

```diff
@@ -53,21 +53,76 @@ public function listAction($offset = 0) {
 			$imageItems = $this->fileCollectionService->getFileObjectsFromCollection($collectionUids);
 			$cObj = $this->configurationManager->getContentObject();
 			$currentUid = $cObj->data['uid'];
+			$columnPosition = $cObj->data['colPos']; //To determine where the gallery is placed (e.g. sidebar)
+
+			//If more than one FileCollection is selected
+			if (sizeof($collectionUids) > 1) {
+				//if a special gallery is requested
+				if ($this->request->hasArgument('galleryUID')) {
+					$gallery = array($this->request->getArgument('galleryUID'));
+					$imageItems = $this->fileCollectionService->getFileObjectsFromCollection($gallery);
+					$showBackToGallerySelectionLink = true;
```

Comment:
Scrutinizr complains about $showBackToGallerySelectionLink not beeing set in any possible outcome, so it needs to be set to false by default or on the else path.